src/services/measurementSystem.ts

```
@@ -0,0 +1,628 @@
+      spaceBelow: position.y
+    };
+  }
+
+  private calculateSpaceToOtherObjects(
+    position: THREE.Vector3,
+    width: number,
+    depth: number,
+    height: number,
+    excludeItemId: number
+  ): { left: number; right: number; front: number; back: number } {
+    let minLeft = Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)